README.md

```diff
@@ -474,3 +474,74 @@ private void ValidateStudentId(Student student) { ... }
 private void ValidateStudentStrings(Student student) { ... }
 private void ValidateStudentDates(Student student) { ... }
 ```
+
+## **Invalid Enum validations**
+
+Let's assume our Student model looks as follows:
+
+```cs
+public class Student {
+	StudentType Type {get; set;}
+}
+
+public enum StudentType{
```

Comment:
@viralpandya push the open bracket to the next line 
do the same for the above

Reply:
Resolved